build_library/grub.cfg

```diff
@@ -72,15 +72,20 @@ if [ -n "$first_boot" ]; then
     set first_boot="coreos.first_boot=1 coreos.randomize_disk_guid=00000000-0000-0000-0000-000000000001"
 fi
 
+set oem=""
 if [ -n "$oem_id" ]; then
-    set oem_id="coreos.oem.id=$oem_id"
+    set oem="coreos.oem.id=$oem_id"
 fi
 
 # If no specific console has been set by the OEM then select based on
 # platform, most systems use vga text as primary and ttyS0 as secondary.
 if [ -z "$linux_console" ]; then
     if [ "$grub_platform" = pc ]; then
-        set linux_console="console=ttyS0,115200n8 console=tty0"
+        if [ "$oem_id" = ec2 ]; then
+            set linux_console="console=ttyS0,115200n8"
```

Comment:
Why are you not just setting `set linux_console="console=ttyS0,115200n8 console=tty0"` in the oem grub.cfg?